.travis.yml

```diff
@@ -12,6 +12,12 @@ rvm:
 before_install:
   - "gem install bundler"
 
+jobs:
+  include:
+    - stage: spec:live
+      rvm: 2.3
+      script: bundle exec rake clean_live set_travis_credentials spec:live
```

Comment:
I decided not to add the spec:live to the common script and not to run it with all rubies for two main reasons:

- It will cause the build to be way slower
- It will cause a few race conditions since we would then have multiple request running in parallel performing the same operations in the same instance.

I think it's fair to assume we can run only one set of these tests. though maybe it we should use the latests available ruby instead of the latest version we support? thoughts?

Reply:
I think if we're going to run only one set of live tests, it should be with the latest Ruby version (2.6.4). People on 2.3 will ultimately upgrade to 2.6, but nobody on 2.6 is going back to 2.3.
We should update travis to use 2.6.4 instead of 2.6.1 while we're at it.